atlascharts/trellisline.js

```diff
@@ -219,7 +241,7 @@ define(["d3", "./chart"],
 	    }
 
 	    // calculate an intial width and height that does not take into account the tick text dimensions
-	    let width = w - options.margins.left - yAxisLabelWidth - options.margins.right;
+	    let width = w - options.margins.left - yAxisLabelWidth - options.margins.right*3;
```

Comment:
What is the magic of `3` that you are multiplying the right margin with?

Reply:
We need to increase right margin of our svg container, so that tooltip values can be fully visible(as they are now positioning to the right from the circle)

Reply:
But if the `options.margins.right` = 0, you won't being increasing anything. Shouldn't it be adding a fixed amount?

Alternatively, couldn't `text-anchor:end` be applied for the last element, and the next will be fully visible?